### PR TITLE
DC: rewrote committees to hopefully skip bad committees

### PR DESCRIPTION
**Pull request by rshorey**
Gets rid of some weird naming, and hopefully will skip bad committees? None seem to be up there now, so can't be 100% sure on that.

### CODE REVIEW

**Comment by jalandis** [comment]
I was looking into this before I noticed your PR.  Just in case it hasn't been noted elsewhere, the bug depends on the date with the XPath selector picking up event links from the calendar as valid committees.

For example, on March 25th 2015, the following would be picked up with the old XPath:
```
<a href="http://dccouncil.us/events/committee-on-education-public-hearing-on-b21-5-b21-31">Committee on Education Public Hearing on B21-5  &amp; B21-31</a>
```

File: openstates/dc/committees.py [hunk 1]
```diff
@@ -13,32 +13,25 @@ def scrape(self, term, chambers):
...
-                com.add_member(chair.text_content(), role='chairperson')
-                chairs.append(chair.text_content())
-
-            for member in doc.xpath('//h3[text()="Councilmembers"]/following-sibling::ul//a'):
-                if member.text_content() not in chairs:
-                    com.add_member(member.text_content(), role='member')
-
-            com.add_source(url)
-            self.save_committee(com)
+        comms = doc.xpath('//li[contains(@class,"node_committee-on")]/a')
```
Comment (mileswwatkins) [inline openstates/dc/committees.py]: The ol' DC misspell causes this scraper rewrite to miss Health and Human Services, whose `class` is `"node_committe-on-health-and-human-services level_2"`.

**Comment by mileswwatkins** [comment]
Otherwise, looks good to go!

**Comment by rshorey** [comment]
committe(e) spelling fixed, @mileswwatkins, ready to merge?

**Comment by mileswwatkins** [comment]
:shipit:

File: openstates/dc/committees.py [hunk 2]
```diff
@@ -13,32 +13,28 @@ def scrape(self, term, chambers):
...
-                chairs.append(chair.text_content())
-
-            for member in doc.xpath('//h3[text()="Councilmembers"]/following-sibling::ul//a'):
-                if member.text_content() not in chairs:
-                    com.add_member(member.text_content(), role='member')
-
-            com.add_source(url)
-            self.save_committee(com)
+        #dc spelled committe(e) two different ways
+        #IN THEIR HTML CLASS NAMES!
```
Comment (paultag) [inline openstates/dc/committees.py]: hilarious

:cries softly in corner:

**Pull request merged**